packages/data-grid/src/tableFooter/TableFooter.jsx

```diff
@@ -129,7 +131,18 @@ function TableFooter({
   pageCount,
   updatePage,
   delta,
+  rowsPerPageOptions,
 }) {
+  const rowsPerPageOptionValues =
+    rowsPerPageOptions.length > 0
+      ? rowsPerPageOptions
+      : [
+          { value: 5, label: '5' },
+          { value: 10, label: '10' },
+          { value: 25, label: '25' },
+          { value: 50, label: '50' },
+        ];
```

Comment:
Could you move this to be defined in `defaultProps`? I'm pretty sure that's how the default values shown in design system website get populated.

Reply:
The defaultProps are used to populate the props table at the top of the design library page - https://puppet.style/#/Data%20Grid/TableFooter
Adding this into the default props will not be formatted correctly for the page. I tried this and it try's to force it into the space.